lib/adf_info.py

```diff
@@ -567,6 +616,62 @@ def add_diag_var(self, var_str):
             self.__diag_var_list.append(var_str)
         #End if
 
+    #########
+
+    #Utility function to grab climo years from pre-made time series files:
+    def get_climo_yrs_from_ts(self, input_ts_loc, case_name):
+        """
+        Grab start and end climo years if none are specified in config file
+        for pre-made time series file(s)
+
+        Return
+        ------
+          - start year
+          - end year
+        """
+
+        #Grab variable list
+        var_list = self.__diag_var_list.copy()
```

Comment:
I _think_ you can use the standard `diag_var_list` property here:

```suggestion
        var_list = self.diag_var_list
```